src/state/commons/DatasetConstants.js

```diff
@@ -24,3 +25,8 @@ export const DATASET_TWINGRAPH_QUERIES_RESULTS_ACTIONS = {
   PROCESS: 'PROCESS_DATASET_TWINGRAPH_QUERY_RESULT',
   RESET: 'RESET_DATASET_TWINGRAPH_QUERIES_RESULTS',
 };
+export const DATASET_PERMISSIONS_MAPPING = {
+  viewer: ['read', 'read_security'],
+  editor: ['read', 'read_security', 'write'],
+  admin: ['read', 'read_security', 'write', 'write_security', 'delete'],
+};
```

Comment:
This mapping is duplicated, can you import it from _src/services/config/ApiConstants.js_ instead please?

Reply:
In fact, this mapping is useful because I already use the config Api constant mapping.